src/common/AndroidUtils.ts

```diff
@@ -458,11 +458,14 @@ export class AndroidUtils {
                         );
                     });
                     child.stderr.on('data', (data) => {
-                        reject(
-                            new Error(
-                                `Could not create emulator. Command failed: ${createAvdCommand}\n${data}`
-                            )
-                        );
+                        const msg = data.toString() as string;
```

Comment:
Just a minor point of a philosophical question here: would it make more sense to type `data` above (to `Buffer`), instead? It feels like that's where the type inference falls down, more than casting to `string` here.